config/routes.rb

```diff
@@ -6,4 +6,6 @@
   get "/login", to: "sessions#new", as: :login
   delete "/logout", to: "sessions#destroy", as: :logout
   resources :links
+  get '/:short_url' => 'links#process_url'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.